cli/src/main/java/com/liferay/blade/cli/Extensions.java

```diff
@@ -223,7 +228,9 @@ public void close() throws IOException {
 
 	public Path getTemplatesPath() throws IOException {
 		if (_embeddedTemplatesPath == null) {
-			_embeddedTemplatesPath = Files.createTempDirectory("templates");
+			long pid = ProcessesUtil.getCurrentProcessId();
+
+			_embeddedTemplatesPath = Files.createTempDirectory("templates-" + pid + "-");
```

Comment:
Lets add a prefix called `blade` to the temp directories.  So `blade-templates` or `blade-extensions`